### PR TITLE
Add PyCon India 2021 theme

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- Adds PyCon India 2021 theme 
- Redirects navbar page to 2021 website
- Assets to be added.

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@palnabarun @Sc0rpi0n101 need a review and a merge, folks.

**Comment by vipulgupta2048** [comment]
@pythonindia/pycon-india-2021-leads-approvers ^

**Comment by vipulgupta2048** [comment]
The PR has been opened for more than a month and needed the merge urgently. Since the header of the blog, still shows the year as 2020 with old PyCon India 2020 pages being linked to it. Will merge the PR on Monday 1630 UTC if there are no further comments to update the blog at least still the new design gets here. 

Screenshot from 2021-04-04 22-38-03

**Review by raukadah — APPROVED** [review]

**Pull request merged**
